mockserver-core/src/test/java/org/mockserver/server/initialize/ExpectationInitializerLoaderTest.java

```diff
@@ -477,11 +477,11 @@ public void shouldLoadExpectationsFromFileSystemInJsonWithGlobWithSubPatterns()
                 )
             };
             File mockserverInitializerThree = File.createTempFile(uniquePrefix + "_mockserverInitializationThree", ".json");
-            Files.write(mockserverInitializerThree.toPath(), expectationSerializer.serialize(expectionsThree).getBytes(StandardCharsets.UTF_8));
+            Files.write(mockserverInitializerThree.toPath(), expectationSerializer.serialize(expectationsThree).getBytes(StandardCharsets.UTF_8));
             ConfigurationProperties.initializationJsonPath(mockserverInitializer.getParentFile().getAbsolutePath() + "/" + uniquePrefix + "_mockserverInitialization{One,Two}*.json");
 
             // when
-            final Expectation[] expectations = new ExpectationInitializerLoader(configuration(), new MockServerLogger(), mock(RequestMatchers.class)).loadExpectations();
+            expectations = new ExpectationInitializerLoader(configuration(), new MockServerLogger(), mock(RequestMatchers.class)).loadExpectations();
```

Comment:
And this conflicts w/ the previous set of `expectations`, and again doesn't really care about the earlier ones.